index.html

```diff
@@ -7,7 +7,184 @@
+              <div class="d-flex">
+                <a href="profile.html" class="text-decoration-none text-dark"
+                  ><img
+                    src="/images/profileImages/profile01.jpg"
+                    alt="user avatar"
+                    class="post-avatar rounded-circle" />
+                </a>
+                <div class="d-flex flex-column ms-2 mt-1">
+                  <h5 class="mb-0">John Denver Smithsonian</h5>
+                  <p class="card-text">
+                    <small class="text-muted">Posted 3 mins ago</small>
+                  </p>
+                </div>
+              </div>
+              <!-- Dropdown start -->
+              <div class="dropdown">
+                <a
+                  class=""
+                  href="#"
+                  role="button"
+                  data-bs-toggle="dropdown"
+                  aria-expanded="false">
+                  <i class="bi bi-three-dots"></i>
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Hide post</a></li>
+                </ul>
+              </div>
+            </div>
+            <!-- Dropdown end -->
+            <div class="card-body">
+              <p class="card-text">
+                This is a wider card with supporting text below as a natural
+                lead-in to additional content. This content is a little bit
+                longer.
+              </p>
+              <div class="d-flex">
+                <p class="bi bi-hand-thumbs-up me-4">&nbsp;Like</p>
+                <p class="bi bi-chat-dots">&nbsp;Comment</p>
+              </div>
+              <div class="input-group">
+                <label for="text"></label>
+                <input
+                  style="border-radius: 25px 0 0 25px"
+                  type="text"
+                  class="form-control bg-secondary bg-opacity-10"
+                  placeholder="Add your comment.." />
+                <button class="btn" style="border-radius: 0 25px 25px 0">
+                  Post
+                </button>
+              </div>
+            </div>
+          </div>
+          <!-- Cards end -->
+          <!-- Load more start -->
+          <div class="d-flex justify-content-center">
+            <button
+              class="btn rounded-pill ps-4 pe-4 shadow mt-4 mb-4"
+              type="load">
+              Load more ..
+            </button>
+          </div>
+          <!-- Load more end -->
+        </div>
+      </div>
+    </div>
+    <!-- Feed & card container end -->
+    <!-- Footer start -->
+    <footer><div>2022 &copy;</div></footer>
+    <!-- Footer end -->
+    <!-- Bootstrap JS -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js"
+      integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3"
+      crossorigin="anonymous"></script>
+    <script
```

Comment:
If you use bootstrap.bundle you don't need two imports here.

Reply:
Done. Imported bundle instead.